crates/swc_ecma_transforms_compat/src/es2015/template_literal.rs

```diff
@@ -9,13 +10,24 @@ use swc_ecma_utils::{
 };
 use swc_ecma_visit::{as_folder, noop_visit_mut_type, Fold, VisitMut, VisitMutWith};
 
-pub fn template_literal() -> impl Fold + VisitMut {
-    as_folder(TemplateLiteral::default())
+pub fn template_literal(c: Config) -> impl Fold + VisitMut {
+    as_folder(TemplateLiteral {
+        c,
+        ..Default::default()
+    })
 }
 
 #[derive(Default)]
 struct TemplateLiteral {
     added: Vec<Stmt>,
+    c: Config,
+}
+
+#[derive(Debug, Clone, Copy, Default, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Config {
+    pub ignore_to_primitive: bool,
+    pub mutable_template: bool,
```

Comment:
I think `#[serde(default)]` is required for config fields.

Reply:
Should existing transformers with options(like `for_of`) be added?